__mocks__/axios.js

```diff
@@ -0,0 +1,61 @@
+const axios = jest.genMockFromModule('axios');
```

Comment:
jest understands that files in that directory are used as a mock instead of actual third-party modules.